domain_models/views.py

```diff
@@ -0,0 +1,30 @@
+"""Contextual view module."""
+from domain_models import models
+
+
+class ContextView(object):
+    """Contextual view class."""
+
+    __model__ = None
+
+    def __init__(self, model):
+        """Model validation.
+
+        :param model: DomainModel
+        """
+        if not issubclass(self.__model__, models.DomainModel):
```

Comment:
This logic could be moved in metaclass. Moving it to metaclass will give a chance to not execute it every time when context is created.